### ISSUE TITLE
Explain project status in README.md

### ISSUE THREAD

**Issue by buhtz** [yo8192/fcron#34]
Dear maintainer,
I appreciate your efforts.

Could you please describe the status and plans of the project in the README.md. This would make it easier for potential contributors decide about contributions and it also helps downstream maintainers and other depending projects make their decisions.

Thanks in advance,
Christian

**Comment by yo8192** [comment]
Good point - I've added one now. Let me know if I missed any important information you'd like to see there.
